src/components/PlanterQuote.js

```diff
@@ -75,15 +65,17 @@ function PlanterQuote(props) {
           alignItems: 'center',
         }}
       >
-        <Avatar
+        <ProfileAvatar
           src={photo}
+          rotation={planter.image_rotation}
           sx={{
             zIndex: '1',
             width: [90, 180],
             height: [90, 180],
             filter: 'drop-shadow(0px 10px 20px rgba(0, 0, 0, 0.25))',
             order: reverse ? 1 : 0,
             ml: reverse ? [26 / 8, 26 / 4] : 0,
+            mt: 'none',
```

Comment:
`'none'` is an invalid value for margin, it should be `0`

Reply:
it is fixed.